osidb/api_views.py

```diff
@@ -828,3 +831,47 @@ class TrackerView(ModelViewSet):
     filterset_class = TrackerFilter
     http_method_names = get_valid_http_methods(ModelViewSet, excluded=["delete"])
     permission_classes = [IsAuthenticatedOrReadOnly]
+
+
+@include_meta_attr_extend_schema_view
+@include_exclude_fields_extend_schema_view
+@extend_schema_view(
+    list=extend_schema(
+        description="List existing alerts for all models.",
+        parameters=[
+            OpenApiParameter(
+                "name",
+                type=OpenApiTypes.STR,
+                location=OpenApiParameter.QUERY,
+                description=(
+                    "Retrieve only Alerts with the specified name, which is given by the "
+                    "model's validation process."
+                ),
+            ),
+            OpenApiParameter(
+                "owner_uuid",
+                type=OpenApiTypes.STR,
+                location=OpenApiParameter.QUERY,
+                description=(
+                    "Retrieve only Alerts related to a model with the given UUID."
+                ),
+            ),
+            OpenApiParameter(
```

Comment:
If my suggestion with the owner model choices will be added I think this might be removed and drf-spectacular will pick up the choices from filter itself.